minitests/litex/nexys_video_sata/Makefile

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+SHELL = bash
+
+SYNTH ?= vivado
+YOSYS = $(XRAY_DIR)/third_party/yosys/yosys
+PART = xc7a200tsbg484-1
+PROJECT_NAME = sata
+
+clean:
```

Comment:
The first target in a Makefile is the default target.  I don't think `clean` is a good default target.  Recommend putting the `all` target first?  If target ordering matters, you can use `.DEFAULT_GOAL` (see https://www.gnu.org/software/make/manual/html_node/Special-Variables.html#Special-Variables)